### PR TITLE
GH-1 - initial bootstrap of java project

### PR DESCRIPTION
**Pull request by noebrito**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Java CI
```
Comment (noebrito) [inline .github/workflows/ci.yml:1]: This file adds basic GitHub Actions CI.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'java'
+    id 'checkstyle'
```
Comment (noebrito) [inline build.gradle:3]: checkstyle is a common java linter.

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    mavenCentral()
+}
+
+dependencies {
+
+    // logging
+    implementation 'ch.qos.logback:logback-classic:1.2.3'
+
+    // api
+    implementation 'io.javalin:javalin-bundle:3.12.0'
```
Comment (noebrito) [inline build.gradle:30]: api framework we'll be using.

**Review by chrisjm — APPROVED** [review]
LGTM 👍  (though, TBH, I'm probably the last person who should be reviewing this 😆 )

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,38 @@
+plugins {
```
Comment (noebrito) [inline build.gradle:1]: Gradle is the build tool we'll be using.

File: build.gradle [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+}
+checkstyleTest {
+    source ='src/test/java'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
```
Comment (noebrito) [inline build.gradle:24]: Application dependencies will be defined here.

File: src/main/java/AppServer.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import io.javalin.Javalin;
```
Comment (noebrito) [inline src/main/java/AppServer.java:1]: This and the next file are the two I actually created.

**Comment by noebrito** [comment]
@chrisjm I just added some comments to the PR as an FYI of what some things are.

File: build.gradle (hunk 4 above)
Comment (noebrito) [reply build.gradle:1]: This file is where we specify how to build the app, declare dependencies, etc.

**Pull request merged**